lib/config.js

```diff
@@ -154,6 +154,8 @@ function getLocalConfig(thisConfig, directory) {
 
         if (personalConfig) {
             return ConfigOps.merge(config, personalConfig);
+        } else if (thisConfig.options.rules && thisConfig.options.rules !== {}) {
```

Comment:
This comparison will always return `true`. The `{}` creates an entirely new object, so `thisConfig.options.rules` can never equal `{}`. What is it you are trying to test for here?